src/main/python/prunetags/cli.py

```diff
@@ -401,6 +408,27 @@ def get_prerelease_tags_report(data, shared_data):
     return report
 
 
+@queue_handler
+def remove_version_tags(data, shared_data):
+    logger.debug(f'@@DATA: --- {data}')
+    logger.debug(f'%%SHARED DATA: --- {shared_data}')
```

Comment:
it would be nice to have the CLI validate version prior to invoking the API, that way if the user enters an invalid expression for the version the CLI would catch it immediately as opposed to having the API fail.